app/controllers/application_controller.rb

```diff
@@ -1,5 +1,26 @@
 class ApplicationController < ActionController::Base
-  # Prevent CSRF attacks by raising an exception.
-  # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+
+  def current_user
+    current_user ||= User.find_by(id: session[:user_id]) if session[:user_id]
+  end
+  helper_method :current_user
+
+  def authorized?
+    params[:id].to_i == session[:user_id]
```

Comment:
We discussed this. Think about having something on the models to determine whether user a can edit object b